tests/perf/overlays/s2n-quic/quic/s2n-quic-core/src/inet/checksum.rs

```diff
@@ -0,0 +1,482 @@
+// Copyright Kani Contributors
```

Comment:
It's problematic to turn off those tests in our regressions because as soon as a new version of Kani is released, their CI is going to break.

@tautschnig AFAIK, the stabilization plan for this feature includes reducing only making this check if necessary. Thus, I think we should put it under a flag for the time being because the flag is eventually going to be removed once the feature is stabilized.

Reply:
@zhassan-aws I have put the check behind the unstable flag for now